base/unicode/checkstring.jl

```diff
@@ -20,11 +20,13 @@ const UTF_SURROGATE = 32        ##< surrogate pairs present
 ## Get a UTF-8 continuation byte, give error if invalid, return updated character value
 @inline function get_continuation(ch::UInt32, byt::UInt8, pos)
     if !is_valid_continuation(byt)
-        throw(UnicodeError(UTF_ERR_CONT, pos, byt))
+        throw(UnicodeError(ERR_CONT, pos, byt))
     end
     (ch << 6) | (byt & 0x3f)
 end
 
+export unsafe_checkstring, checkstring
```

Comment:
Isn't the convention to export all functions from the same place instead of where they are defined?